src/components/GalleryPage/Winners.js

```diff
@@ -74,35 +75,41 @@ function Winners() {
 	}
 
 	const winnerCards = winners.map(item => {
+		const image = winnerImageMap.get(item.id);
 		return (
-			<Grid container key={item.title} spacing={2} alignItems='center'
-				style={{ paddingTop: 60 }}>
-				<Grid item md={5} xs={12}>
-					<Img fluid={winnerImageMap.get(item.id)} />
-				</Grid>
-				<Grid item md={7} xs={12}>
-					<Typography variant='h5'>{item.title} –{' '}
-						<span style={{ fontStyle: 'italic' }}>
-							{item.category}
-						</span>
+			<Grid item key={item.title} sm={6} style={{ paddingBottom: isSmall ? theme.spacing(4) : theme.spacing(8) }}>
+				<Box display='flex' flexDirection='column'>
```

Comment:
Is it possible to put the styles on the Grid item, rather than making a new Box?